raytracing/specialtylenses.py

```diff
@@ -205,6 +205,156 @@ def pointsOfInterest(self, z):
         return [{'z': f1, 'label': '$F_f$'}, {'z': f2, 'label': '$F_b$'}]
 
 
+class SingletLens(MatrixGroup):
```

Comment:
A Singlet lens cannot be achromatic: the index of refraction is never constant, and obviously the radii of curvature will not change.  Therefore, the lens has to suffer from chromatic aberrations, that's why it is called just 'Singlet'.  The AchromaticDoublet solve that problem partially with two materials.

Reply:
Confirm you have seen this, then I will merge.

Reply:
I already knew that but it slipped my mind. Indeed, SingletLens makes much more sense than AchromaticSingletLens.